packages/utils/src/TestUtils.test.tsx

```diff
@@ -19,6 +19,42 @@ describe('asMock', () => {
   expect(someFunc('a,b,c')).toEqual(3);
 });
 
+describe('disableConsoleOutput', () => {
+  const arg0 = 'mock arg';
+  const allMethodNames = ['log', 'warn', 'error', 'info', 'debug'] as const;
+
+  it('should disable all methods if given no method names', () => {
+    TestUtils.disableConsoleOutput();
+
+    allMethodNames.forEach(methodName => {
+      // eslint-disable-next-line no-console
```

Comment:
You can just wrap the entire test block w/ this instead

```
/* eslint-disable no-console */
// Code here that will have no-console turned off
/* eslint-enable no-console */
// Code here will lint no-console rule again
```

Reply:
I'm aware of that format, but still ends up being 2 eslint comments. Didn't really see a benefit for this particular use case

Reply:
actually nvm. I see there were more